peewee.py

```diff
@@ -4413,6 +4413,14 @@ def bind(self, model, name, set_attribute=True):
     def column(self):
         return Column(self.model._meta.table, self.column_name)
 
+    @property
+    def has_default(self):
+        if self.constraints:
+            for constraint in self.constraints:
+                if constraint.sql.startswith("DEFAULT"):
```

Comment:
probably should make this case insensitive just to be safe, since something like `SQL('default null')` would be valid too